Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -568,19 +576,19 @@
     "    name=name,\n",
     "    description=\"An end to end mnist example including hyperparameter tuning, train and inference\"\n",
     ")\n",
-    "def mnist_pipeline(name=name, namespace=namespace, training_steps=training_steps):\n",
+    "\n",
+    "def mnist_pipeline(name: str, namespace: str):\n",
     "\n",
     "    # Run the hyperparameter tuning with Katib.\n",
     "    katib_op = create_katib_experiment_task(name, namespace, training_steps)\n",
-    "\n",
-    "    volume_str = f\"model-volume-{uuid}\"\n",
-    "    \n",
+    "    # Getting the storage class\n",
+    "    storage_class_name = !kubectl get pvc user-pvc -o jsonpath={.spec.storageClassName}\n",
     "    # Create volume to train and serve the model.\n",
-    "    model_volume_op = dsl.VolumeOp(\n",
-    "        name=volume_str,\n",
-    "        resource_name=volume_str,\n",
+    "    model_volume_op = kfp.kubernetes.CreatePVC(\n",
+    "        pvc_name=volume_str,\n",
     "        size=\"1Gi\",\n",
-    "        modes=dsl.VOLUME_MODE_RWO\n",
+    "        storage_class_name=storage_class_name[0],\n",
```

Comment:
Do we need to pass a storage class name? I think it is not required, and if we omit it here - default storage class will be used by kubernetes

Reply:
Without storage_class_name parameter, PVC was getting created in `standard` storage_class, since this storage_class is not present with this name, therefore, PVC and the TF jobs were going in Pending state. Same was seen when we tried with empty ("") storage_class_name.

Reply:
Hmm, got it, okay, Than yes, we can leave it as is